habitat-lab/habitat/core/batch_rendering/env_batch_renderer.py

```diff
@@ -240,13 +275,39 @@ def copy_output_to_image(self) -> List[np.ndarray]:
 
         :return: List of RGB images as ndarrays.
         """
-        # TODO: Only one color sensor supported.
+        # TODO: Only one sensor supported.
         output: List[np.ndarray] = []
         if self._gpu_gpu:
             raise NotImplementedError
         else:
+            sensor_spec = self._sensor_specifications[0]
             for env_idx in range(self._num_envs):
-                output.append(self._gpu_to_cpu_buffer[env_idx][..., 0:3])
+                if sensor_spec.sensor_type == habitat_sim.SensorType.COLOR:
```

Comment:
How about a helper function to encapsulate this depth transformation? (And then maybe in the future it gets reused here and in the classic-renderer codepath.)